src/main/java/org/apache/maven/plugin/patch/ApplyMojo.java

```diff
@@ -1,3 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
```

Comment:
I know about planned plugin retirement, but can't resist: too bad the license text from below was not deleted...

Reply:
@pzygielo What is wrong with this licenses ... I don't see ...

Can you create a PR?

Reply:
> @pzygielo What is wrong with this licenses ... I don't see ...

Nothing. But the license text is twice now - the newly added before `package`, and the old one after `package`.

![image](https://github.com/user-attachments/assets/06f50dbe-952c-49e4-916c-e8769f431e09)



> Can you create a PR?

- #6